packages/shared/components/NetworkIconBadge.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="typescript">
+    import { NetworkIcon, Text, Tooltip } from 'shared/components'
+    import { NETWORK, NetworkProtocol, NetworkType } from '@core/network'
+
+    export let networkType: NetworkType
+    export let networkProtocol: NetworkProtocol
+
+    let tooltipAnchor
+    let showTooltip = false
+
+    $: tooltipText = NETWORK[networkProtocol]?.[networkType]?.name
```

Comment:
and optional chain in here just in case
```suggestion
    $: tooltipText = NETWORK?.[networkProtocol]?.[networkType]?.name
```